backend/src/controllers/item.js

```diff
@@ -131,6 +132,58 @@ const ItemController = (itemModel, userModel, authService) => {
     });
   });
 
+  // Gets the logged in user using the authentication token passed in the request header
```

Comment:
Even though we arent doing JSDoc I still think we should do good comments on these methods, explaining what they do. This one seems out of place